openmdao/recorders/case.py

```diff
@@ -0,0 +1,221 @@
+"""
+A Case class.
+"""
+
+
```

Comment:
Not sure how Case and its children passed lint. Params and attributes don't match the code in __init__.